routers/transactions.controller.js

```diff
@@ -19,19 +19,6 @@ class TransactionController {
 
   async createTransaction(req, res) {
     try {
-      const user = await userModel.findById(req.user);
```

Comment:
You need to check here at least that `req.user.familyId` is present

Reply:
I check it in middleware familyAuthorization before calling this method, so if there is no familyId, it woun't get here, as I understand